boards/xtensa/up_squared_adsp/tools/fw_loader/stream_desc.py

```diff
@@ -0,0 +1,233 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2018 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+

```

Comment:
This file seems to screwed up in text editor, I see:
```
+#!/usr/bin/env python3^M
+#^M
+# Copyright (c) 2018 Intel Corporation^M
+#^M
+# SPDX-License-Identifier: Apache-2.0^M
+^M
```

Reply:
Fixed it.